examples/upgradable-examples/upgradable_base/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+RUSTFLAGS='-C link-arg=-s' cargo build --features "counter2" --target wasm32-unknown-unknown --release
+mv  ../../target/wasm32-unknown-unknown/release/upgradable_base.wasm ../../target/wasm32-unknown-unknown/release/upgradable_base_second.wasm
+RUSTFLAGS='-C link-arg=-s' cargo build --features "counter1" --target wasm32-unknown-unknown --release
```

Comment:
It's better to store both files in the `examples/res` directory.

Reply:
If you run the examples/build.sh script the both files will be stored in examples/res directory. 

examples/build.sh runs all build.sh scripts and copy all the wasm files into res directory. 

Do you think I should copy wasm files to res dir in all build.sh scripts?

Reply:
Yes, if you use this script, it should copy to `res` dir. The top-level script could rather execute each example script. This way, you can have a more clean approach for the `Upgradable` contract, for example, which will utilize the file from the `res` directory.

Reply:
fixed: https://github.com/aurora-is-near/near-plugins/pull/34/commits/5c8f7548888eaa6f0c2d2cb10b28604a1e9c9c49